src/utils/testing.utils.ts

```diff
@@ -18,6 +20,9 @@ export function fakeResponse(overrides?: Partial<Response>): Response {
   res.json = jest.fn().mockImplementation((r: any) => res)
   res.send = jest.fn().mockImplementation(() => res)
   res.end = jest.fn().mockImplementation(() => res)
+  res.cookie = jest.fn().mockImplementation((n: string, v: string, o: CookieOptions) => res)
+  res.redirect = jest.fn().mockImplementation(() => res)
+  res.type = jest.fn().mockImplementation(() => res)
```

Comment:
Would it make sense to generate mocks in one place (here or separate files) for our other types/etc that get used often in tests (User, etc) and reference them instead of duplicating for sake of updating in one place for future changes?

Reply:
I think regardless you're gonna have want to have to invoke a thing since you should always be creating a new mock on a per test basis. if we wanted to make some test thing to do this that'd be fine, but I think it'd just obfuscate some stuff that we probably want a bit more explicit.